packages/wrangler/src/dev.ts

```diff
@@ -600,6 +601,20 @@ export async function startDev(args: StartDevOptions) {
 			logger.loggerLevel = args.logLevel;
 		}
 
+		if (args.remote) {
+			const config = readConfig(args);
+
+			if (args.assets || config.assets) {
+				const ok = await confirm(
+					"Using `wrangler dev --remote` with assets requires a new Worker to be uploaded to the server. Do you want to continue?"
```

Comment:
What does this actually mean? `uploaded to the server` doesn't tell me much. Is it just for new sessions? Will it deploy the current state of the codebase to production?

Reply:
yeah, the copy here prolly needs some rephrasing. I basically took what I had in the ticket. Maybe @korinne or @GregBrimble want to chime in here?